docs/modules/user-guide/partials/proc_deploying-a-devfile-registry-with-olm.adoc

```diff
@@ -0,0 +1,55 @@
+[id="deploying-a-devfile-registry_{context}"]
+= Deploying a devfile registry with operator lifecycle manager
+
+[role="_abstract"]
+When you have operator lifecycle manager (OLM) installed on your cluster, use the devfile registry operator to install and deploy a devfile registry.
+
+.Procedure
+..Installing the devfile registry operator
+. Install link:https://github.com/operator-framework/operator-lifecycle-manager[OLM].
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/operator-sdk/latest/[operator-sdk CLI].
+. Run `operator-sdk run bundle quay.io/devfile/registry-operator-bundle:next` to install the operator-sdk CLI.
```

Comment:
This can't be right, can it? We run the `operator-sdk` command to install the `operator-sdk` command line interface? That doesn't seem correct, but I don't know enough about this procedure to comment on what would be.

Regardless of that, however, I would include the command itself in a block:

```adoc
. Install the Operator SDK CLI:
+
[subs="+quotes"]
----
$ operator-sdk run bundle quay.io/devfile/registry-operator-bundle:next
----
```

Reply:
Yeah, it should be two separate points:
1. Install the operator CLI
2. Run that command to install the devfile-registry operator.